controllers/dbaas.redhat.com/conditions.go

```diff
@@ -22,15 +22,17 @@ import (
 )
 
 const (
-	SpecSynced          string = "SpecSynced"
-	BackendError        string = "BackendError"
-	AuthenticationError string = "AuthenticationError"
-	SyncOK              string = "SyncOK"
-	ReadyForBinding     string = "ReadyForBinding"
-	Ready               string = "Ready"
-	NotFound            string = "NotFound"
-	SuccessMessage      string = "Successfully listed crunchy bridge Inventories"
-	SuccessConnection   string = "Successfully retrieved the connection detail\n"
+	SpecSynced             string = "SpecSynced"
+	BackendError           string = "BackendError"
+	AuthenticationError    string = "AuthenticationError"
+	SyncOK                 string = "SyncOK"
+	ReadyForBinding        string = "ReadyForBinding"
+	ClusterReady           string = "ClusterReady"
```

Comment:
Is this Condition type? ADR is ```ProvisionReady```?
Do I missing anything? which one we should use?

Reply:
Good catch @myeung18 its `ProvisionReady` I will update it.

Reply:
updated